_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+
+The raw data is available in a JSON format, and is very detailed  with information on shipments from specific coal mines to power plants, and also on the operating statistics of power plants.
+
+# Coal Production
+
+Here we use the [EIA bulk data on coal](http://api.eia.gov/bulk/COAL.zip){:target="_blank"} which [can also be browsed online](https://www.eia.gov/opendata/qb.php?category=717234){:target="_blank"}.
+
+The animation below shows the yearly amount of coal produced, both at the level of individual mines, and also aggregated statistics for total mine production per state.  From this one can see that the mines tend to be locally concentrated based on the [various coal basins](https://www.eia.gov/energyexplained/index.php?page=coal_where){:target="_blank"}, and also the amount of production is by no means evenly distributed.
+
+![Coal Production]({{ site.baseurl }}public/images/CoalProduction.gif)
+<sup>_Yearly coal production for electricity usage, shown at both mine and state level._</sup>
+
+As someone who grew up in the US, this map challenges the common mythology I often heard about [Appalachia and coal mining](https://en.wikipedia.org/wiki/Appalachia#Coal_mining){:target="_blank"}, and assumptions about states such as West Virginia being dominant producers.  While Appalachia was a primary source of coal historically, it has been far eclipsed by Wyoming's [Powder River Basin](https://www.scientificamerican.com/article/powder-rive-basin-coal-on-the-move/){:target="_blank"}.
+
+The Powder River Basin is composed of surface mines with giant coal seams as shown below.  This region produces around 42% of all coal in the US[^PowderRiverBasin_42_percent], with the North Antelope Rochelle Mine alone providing 12% of total US production in 2016[^NorthAntelope_12_percent].  This single mine produces more coal than West Virginia, the second largest coal mining state.
+
+![Coal Seam]({{ site.baseurl }}public/images/Coal_Seam_at_Coal_Mine.png)
+<sup>_Coal seam, [North Antelope Rochelle Mine](https://en.wikipedia.org/wiki/North_Antelope_Rochelle_Mine){:target="_blank"}. By Peabody Energy, Inc. [CC BY 3.0](https://creativecommons.org/licenses/by/3.0){:target="_blank"}, via [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Coal_Seam_at_Coal_Mine.png){:target="_blank"}_</sup>
+
+The animated map below by the [Google Earth Engine Timelapse](https://earthengine.google.com/timelapse/#v=43.60679,-105.24646,9.613,latLng&t=2.96){:target="_blank"} shows the enormous geographic scale of the Power River Basin mines, along with their historical growth over 32 years of satellite imagery.  Over time, one can see new areas being dug out with land restoration efforts following shortly behind.
```

Comment:
--> "It's interesting see new areas being dug out, with land restoration efforts following shortly behind."